pages/portfolio.html

```diff
@@ -1,12 +1,54 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
```

Comment:
This page could be added to your directory pages so that all of your pages are in one place.